test/e2e/performanceprofile/functests/1_performance/performance.go

```diff
@@ -304,6 +304,30 @@ var _ = Describe("[rfe_id:27368][performance]", Ordered, func() {
 				Expect(processesFound).To(BeEmpty(), "The node %s has the following processes on the root cgroup: %v", node.Name, processesFound)
 			}
 		})
+
+		It("Check for agetty process system.slice cgroup", func() {
+			for _, node := range workerRTNodes {
+				agettyPidCommand := "pgrep agetty"
+				agettyPidOutput, err := nodes.ExecCommandOnNode([]string{"/bin/bash", "-c", agettyPidCommand}, &node)
+				Expect(err).ToNot(HaveOccurred())
+				agettyPids := strings.Fields(agettyPidOutput)
```

Comment:
let's expect  agettyPidOutput not to be empty before checking the pids

Reply:
Resolved